src/t8_schemes/t8_default/t8_default_pyramid/t8_default_pyramid_cxx.cxx

```diff
@@ -300,7 +300,7 @@ t8_default_scheme_pyramid_c::t8_element_is_family (t8_element_t **fam)
     T8_ASSERT (t8_element_is_valid (fam[i]));
   }
 #endif
-  return t8_dpyramid_is_family ((const t8_dpyramid_t **) fam);
+  return t8_dpyramid_is_family ((t8_dpyramid_t **) fam);
```

Comment:
Reminder: Check why const is no longer possible here, because is_family should leave the elements const